python/common/org/python/types/ByteArray.java

```diff
@@ -413,33 +413,33 @@ public ByteArray(org.python.Object[] args, java.util.Map<java.lang.String, org.p
             byte[] other_bytes = (byte[]) ((org.python.types.Bytes) other).value;
             for (int i = 0; i < Math.min(this.value.length, other_bytes.length); i++) {
                 if (this.value[i] > other_bytes[i]) {
-                    return new org.python.types.Bool(1);
+                    return org.python.types.Bool.getBool(1);
```

Comment:
`getBool(1)` should have a very predictable output... :-)

Reply:
Done, here and in other places :-)